frontend/src/components/app/scrollTop.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+class ScrollToTopOnMount extends React.Component {
+  componentDidMount() {
+    window.scrollTo(0, 0);
```

Comment:
why is it necessary to scroll the window to the top of the page?

Reply:
you can also add this windows.scrollTo method in the playersComponent's componentDidMount method (React.useEffect hook in Function Components)